### PR TITLE
RickAndMorty - Step 1 Exercise

### PR DESCRIPTION
**Pull request by LetiRocha**
Added button with an clicklistener to change the content of the below text view when tapping the button

### CODE REVIEW

**Comment by PabloGrelaM** [comment]
@coderabbit review

File: app/src/main/java/com/mstudio/superheroarch/MainActivity.kt [hunk 1]
```diff
@@ -16,5 +18,10 @@ class MainActivity : AppCompatActivity() {
             v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
             insets
         }
+        val button = findViewById<Button>(R.id.myfirstbutton)
+        button.setOnClickListener {
+            val textChangedByButton = findViewById<TextView>(R.id.title_changedbybutton)
+            textChangedByButton.text = "The button was clicked!"
```
Comment (PabloGrelaM) [inline app/src/main/java/com/mstudio/superheroarch/MainActivity.kt:24]: You should try to use always R.string.xxx for strings.

**Review by PabloGrelaM — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/mstudio/superheroarch/MainActivity.kt [hunk 2]
```diff
@@ -16,5 +18,10 @@ class MainActivity : AppCompatActivity() {
             v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
             insets
         }
+        val button = findViewById<Button>(R.id.myfirstbutton)
+        button.setOnClickListener {
+            val textChangedByButton = findViewById<TextView>(R.id.title_changedbybutton)
```
Comment (YagoAccenture) [inline app/src/main/java/com/mstudio/superheroarch/MainActivity.kt:23]: Var textChangedByButton need to be created outside the button clicklistener

File: app/src/main/java/com/mstudio/superheroarch/MainActivity.kt (hunk 1 above)
Comment (YagoAccenture) [inline app/src/main/java/com/mstudio/superheroarch/MainActivity.kt:24]: Text should be saved as string

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -8,12 +8,23 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/title_changedbybutton"
```
Comment (YagoAccenture) [inline app/src/main/res/layout/activity_main.xml:11]: All variables should use snake case

**Review by YagoAccenture — CHANGES_REQUESTED** [review]

**Review by YagoAccenture — APPROVED** [review]

**Pull request merged**